roles/wazuh/ansible-wazuh-manager/defaults/main.yml

```diff
@@ -4,6 +4,12 @@ wazuh_manager_version: 4.0.0-1
 wazuh_manager_fqdn: "wazuh-server"
 wazuh_manager_package_state: present
 
+nodejs:
```

Comment:
Since the Wazuh API is already integrated along with the core, Node.JS is not required anymore.

Reply:
Hey manuasir! you're right :smile: . Considering that these changes also modify tasks, how about I tackle them in a separate PR? thanks!